app/controllers/album.js

```diff
@@ -1,13 +1,31 @@
 'use strict';
 
 const errors = require('../errors'),
-  { getAll } = require('../services/album'),
+  { getAlbums } = require('../services/album'),
+  Album = require('../models').albums,
   logger = require('../logger');
 
 exports.albumList = (req, res, next) => {
-  return getAll('/albums')
+  return getAlbums('/albums')
     .then(albums => {
       res.status(200).send(albums);
     })
     .catch(next);
 };
+
+exports.buyAnAlbum = async (req, res, next) => {
+  try {
+    const album = await getAlbums(`/albums/${req.params.id}`);
+
+    const newAlbum = {
```

Comment:
What happen if album is missing  ?

Reply:
the error is catched and the user is informate about de error